src/Ledger/GovernanceActions.lagda

```diff
@@ -178,25 +319,24 @@ private variable
 
 instance
   _ = +-0-monoid
-
-data
```

Comment:
I'd like to keep this part. It looks a lot like the Shelley spec & hides the type theory. We don't want to gatekeep the spec behind an understanding of 'propositions as types'.